django/src/rdwatch/server/settings.py

```diff
@@ -1,63 +1,125 @@
-from os import environ
-from urllib.parse import urlparse
+import os
 
-SECRET_KEY = environ['RDWATCH_SECRET_KEY']
+from configurations import Configuration, values
 
+_environ_prefix = 'RDWATCH'
 
-ALLOWED_HOSTS = ['*']
-DEBUG = environ['RDWATCH_DJANGO_DEBUG'].lower() in ('1', 'true', 'yes', 'on')
```

Comment:
I think we need both of these (probably the DEBUG one modified) inside the DevelopmentConfiguration.  The server returns all 404's if the Allowed_Hosts isn't in there.  I assume probably in DevelopmentConfiguration we would want DEBUG set to one already or we can check for the environment Prefix 'RDWATCH_DJANGO'.

Reply:
Thanks, I think this is why I was having issues in our debug session yesterday.